google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java

```diff
@@ -695,6 +698,23 @@ public HttpRequest setFollowRedirects(boolean followRedirects) {
     return this;
   }
 
+  /**
+   * Return whether to use raw redirect urls. 
+   */
+  public boolean getUseRawRedirectURLs() {
```

Comment:
Sorry, I had decided to let the library team check code style and whatnot, but I must point this out now, as method names are important as a public API.

The Google-style for names is `...Urls`. (Don't get me wrong. You'll write "URL" in Javadoc texts.")

Reply:
@chanseokoh: I am getting contradicting input and I am getting a little bit confused.
@elharo: requested to change the mehtod name from xxxxUrls to xxxxURLs. 

So currently you'd like me to change that to:

      /**
       * Return whether to use raw redirect URLs. 
       */
       public boolean getUseRawRedirectUrls() {
       }

Is that right?

Reply:
Sorry if this is confusing. My request was for the javadoc, not the method name. That is, "use raw redirect urls." --> "use raw redirect URLs."

Reply:
> Is that right?

@iocanel yes, correct.